test/integration/testdata/valuevalidation/alpha.go

```diff
@@ -0,0 +1,72 @@
+package valuevalidation
+
+import (
+	"k8s.io/kube-openapi/pkg/common"
+	"k8s.io/kube-openapi/pkg/validation/spec"
+)
+
+// Dummy type to test the openapi-gen API rule checker.
+// The API rule violations are in format of:
+// -> +k8s:validation:[validation rule]=[value]
+
+// +k8s:validation:maxProperties=5
```

Comment:
need to add k8s:open-api gen tags

and enable test integration_test_suite.go, and add route for it in testuti/testutil.go

This process should be made far simpler in a future PR